spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+RSpec.describe 'Users-show', type: :feature do
+  describe 'users show' do
+    before(:each) do
+      @user_one = User.create! name: 'John', photo: 'photo.svg', bio: 'lorem ipsum', postscounter: 0
```

Comment:
- [ ] in the test for the User show page the below test is missing:
Tests that asserts you can see the number of posts the user has written.